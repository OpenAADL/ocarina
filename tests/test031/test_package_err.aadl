
package Test
public

  data dt
  end dt;

properties
  Source_Data_Size => 1 MByte;
end Test;

package Test2
public

  data dt
  end dt;

private

  data bt
  end bt;

properties
  Source_Data_Size => 1 MByte;
end Test2;

package Test3
public

  data dt
  end dt;

properties
  Source_Data_Size => 1 MByte;

private

  data br
  end br;


end Test3;