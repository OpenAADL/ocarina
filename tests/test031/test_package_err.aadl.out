test_package_err.aadl:9:23:  (signed aadlnumber)  and  (property type)  have incompatible unit types
test_package_err.aadl:24:23:  (signed aadlnumber)  and  (property type)  have incompatible unit types
test_package_err.aadl:34:23:  (signed aadlnumber)  and  (property type)  have incompatible unit types
Cannot analyze AADL specifications
